### DISCUSSION TITLE
Percentile rankings, typing rank bell curve

### DISCUSSION THREAD

**Discussion by sealvessel** [monkeytypegame/monkeytype#2185] [Ideas]
Let us see our typing percentile, with typing IQ score. 

![image](https://user-images.githubusercontent.com/66351949/146344469-6ae951e1-34b7-4cb0-9dd2-56641dc27ac2.png)
![image](https://user-images.githubusercontent.com/66351949/146344517-7200dfc8-d969-484e-a6ba-c459ddea2119.png)

**Comment by prat8897** [comment]
[Keybr.com](Keybr.com) has this, I would like to see this on Monkeytype as well. 
<img width="1103" alt="Screenshot 2021-12-26 at 00 35 23" src="https://user-images.githubusercontent.com/7300465/147391888-213da78a-fdd5-4720-956b-91c15a9effd2.png">
